src/components/atoms/IconStore/IconStore.tsx

```diff
@@ -57,15 +62,23 @@ class IconStore extends React.PureComponent<Props, State> {
 
     } else {
       const MI_NAME = MI_ICON_LIST[iconName as keyof typeof MI_ICON_LIST];
+      let fontSizeStyle = width;
+
+      if (fontSize) {
+        fontSizeStyle = fontSize;
+      }
+
```

Comment:
can we use the ternary operator instead ?

Reply:
Done